dockerfiles/vault-dcos/Readme.md

```diff
@@ -0,0 +1,113 @@
+# Vault on DC/OS
+
+This is an example of how to run [HashiCorp's Vault](https://github.com/hashicorp/vault) on DC/OS. By default, it's configured in HA mode with ZooKeeper as the backend.
+
+## Running on DC/OS
+
+### Step 1: Launch Vault server
+Save the following JSON as `vault.json`:
+```json
+{
+  "id": "vault",
+  "cpus": 1,
+  "mem": 1000,
+  "requirePorts":true,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "geoint/scale-vault",
+      "network": "HOST",
```

Comment:
Need to switch this to bridge networking and specify the internal port. Ref here for example: https://github.com/ngageoint/scale/blob/master/dockerfiles/framework/scale/bootstrap.py#L294